src/main/java/appeng/me/storage/MEInventoryHandler.java

```diff
@@ -38,12 +39,14 @@ public class MEInventoryHandler<T extends IAEStack<T>> implements IMEInventoryHa
     private int myPriority;
     private IncludeExclude myWhitelist;
     private AccessRestriction myAccess;
+    private StorageFilter storageFilter;
     private IPartitionList<T> myPartitionList;
 
     private AccessRestriction cachedAccessRestriction;
     private boolean hasReadAccess;
     private boolean hasWriteAccess;
     private boolean isSticky;
+    private boolean gettingAvailableContent;
```

Comment:
Is there a reason why this is made into a field instead of just within the method? I'm not seeing where this would be necessary unless something is calling from other threads (is that the case?)

Reply:
Blind assumption from looking at the upstream repo that this could cause recursion.